docs/pages/resources/fundamentals-libraries.md

```diff
@@ -0,0 +1,22 @@
+---
+title: SAP Fiori Fundamentals Libraries
+keywords: libraries, frameworks
+sidebar: left-navigation-sidebar
+toc: false
+permalink: /fundamentals-libraries/
+folder: resources
+summary: SAP Fiori Fundamentals can be used with your framework of choice, however fundamentals-react, fundamentals-ngx, and fundamentals-vue are currently in active development.
```

Comment:
all links should be done in markdown syntax

Reply:
Okay, so should I not try to open the links in new tabs? Markdown does not allow that.

Reply:
kramdown.